src/Features/CSharp/Portable/Completion/KeywordRecommenders/AbstractSpecialTypePreselectingKeywordRecommender.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Extensions.ContextQuery;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.KeywordRecommenders
+{
+    internal abstract class AbstractSpecialTypePreselectingKeywordRecommender : AbstractSyntacticSingleKeywordRecommender
+    {
+        public AbstractSpecialTypePreselectingKeywordRecommender(SyntaxKind keywordKind, bool isValidInPreprocessorContext = false, bool shouldFormatOnCommit = false) : base(keywordKind, isValidInPreprocessorContext, shouldFormatOnCommit)
```

Comment:
looks to be about 120 chars max :)   That seems like a reasonable wrapping zone.